crates/store/re_video/src/decode/ffmpeg.rs

```diff
@@ -349,6 +355,21 @@ fn read_ffmpeg_output(
         false
     }
 
+    fn log_ffmpeg_warning_once(debug_name: &str, msg: &str) {
+        // Make warn_once work on `[swscaler @ 0x148db8000]` style warnings even if the address is different every time.
+        // In older versions of FFmpeg this may happen several times in the same message (happend in 5.1, did not happen in 7.1).
+        let mut msg = msg.to_owned();
+        while let Some(pos) = msg.find("[swscaler @ 0x") {
+            msg = [
+                &msg[..pos],
+                &msg[(pos + "[swscaler @ 0x148db8000]".len())..],
+            ]
+            .join("[swscaler]");
+        }
```

Comment:
Could be a free-standing function with a unit-test, just to clarify what it is doing